### ISSUE TITLE
Matchers type conflicts with new jest types

### ISSUE THREAD

**Issue by thaapasa** [kbiedrzycki/jest-xml-matcher#19]
Latest jest types from `@types/jest` (24.0.23) conflict with the jest `Matchers` extension, at least on TypeScript 3.7.2.

```
node_modules/@types/jest/index.d.ts:729:15 - error TS2428: All declarations of 'Matchers' must have identical type parameters.

729     interface Matchers<R, T> {
                  ~~~~~~~~

node_modules/jest-xml-matcher/index.d.ts:2:13 - error TS2428: All declarations of 'Matchers' must have identical type parameters.

2   interface Matchers<R> {
              ~~~~~~~~
```

Fix: change `Matchers<R>` to `Matchers<R, T>` in `index.d.ts`.

**Comment by thaapasa** [comment]
Made a PR for this, please merge :)

**Comment by kbiedrzycki** [comment]
Thanks for this update. I'll push relevant updates soon with dependencies bump.

**Comment by kbiedrzycki** [comment]
@thaapasa I've released 1.2.0 that should fix your issue, let me know.
